### PR TITLE
Get rid of v1 execution engine and isolate Spring Batch dependency

### PR DESCRIPTION
**Pull request by robfletcher**
This removes all the legacy code involved in the "v1" execution engine. Now all Spring Batch dependent code is isolated in the `orca-spring-batch` module.

### CODE REVIEW

**Comment by robfletcher** [comment]
PTAL @spinnaker/netflix-reviewers @spinnaker/google-reviewers

File: orca-spring-batch/src/main/groovy/com/netflix/spinnaker/orca/batch/RetryableTaskTasklet.groovy [hunk 1]
```diff
@@ -99,7 +97,8 @@ class RetryableTaskTasklet extends TaskTasklet {
   /**
    * In order to accurately determine whether this task has timed out, any time spent in a paused state must be discounted.
    */
-  private static long determineCurrentExecutionTime(Clock clock, long startTime, PausedDetails pausedDetails) {
+  private
+  static long determineCurrentExecutionTime(Clock clock, long startTime, PausedDetails pausedDetails) {
```
Comment (robzienert) [inline orca-spring-batch/src/main/groovy/com/netflix/spinnaker/orca/batch/RetryableTaskTasklet.groovy]: Unnecessary new line.

Comment (robzienert) [inline orca-spring-batch/src/main/groovy/com/netflix/spinnaker/orca/batch/RetryableTaskTasklet.groovy]: And in a few other places below.

**Comment by robzienert** [comment]
Did a look-through, but I'm not really familiar enough to technically vet it.

File: orca-spring-batch/src/main/groovy/com/netflix/spinnaker/orca/batch/RetryableTaskTasklet.groovy (hunk 1 above)
Comment (robfletcher) [inline orca-spring-batch/src/main/groovy/com/netflix/spinnaker/orca/batch/RetryableTaskTasklet.groovy]: That's IntelliJ auto-wrapping lines, sorry.

**Comment by ttomsu** [comment]
Onlookers FYI: This caused some startup failures in our environment. @robfletcher and I are looking into it.

**Pull request merged**